src/Server/Resource/DoctrineResource.php

```diff
@@ -348,6 +348,11 @@ public function create($data)
 
         $this->getObjectManager()->flush();
 
+        $results = $this->triggerDoctrineEvent(DoctrineResourceEvent::EVENT_CREATE_FLUSH, $entity, $data);
+        if ($results->last() instanceof ApiProblem) {
```

Comment:
OK, I've written in 26132ae631bfda1a8ec8b9c5217df4bae2d50b4c, but I'm not native english speaker, so could you check for language mistakes if any?